lighthouse-core/test/audits/metrics/interaction-to-next-paint-test.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const InteractionToNextPaint = require('../../../audits/metrics/interaction-to-next-paint.js');
+const interactionTrace = require('../../fixtures/traces/timespan-responsiveness-full.trace.json');
+const noInteractionTrace = require('../../fixtures/traces/jumpy-cls-m90.json');
+
+/* eslint-env jest */
+
+describe('Interaction to Next Paint', () => {
+  function getTestData() {
+    const artifacts = {
+      GatherContext: {gatherMode: 'timespan'},
+      traces: {
+        [InteractionToNextPaint.DEFAULT_PASS]: interactionTrace,
+      },
+    };
+
+    const context = {
+      settings: {throttlingMethod: 'devtools'},
+      computedCache: new Map(),
+      options: InteractionToNextPaint.defaultOptions,
+    };
+
+    return {artifacts, context};
+  }
+
+  it('evaluates INP correctly', async () => {
+    const {artifacts, context} = getTestData();
+    const result = await InteractionToNextPaint.audit(artifacts, context);
+    expect(result).toEqual({
+      // TODO: this is incorrect, only looking at main frame events right now.
```

Comment:
the longest responsiveness event is from the child frame, so this will be updated when #13913 lands